### PR TITLE
Added tasks 21-30

### PR DESCRIPTION
**Pull request by KamilWasilonek**

### CODE REVIEW

File: tasks/21_check-letters/check-letters.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export function checkLetters(arr) {
+  for (let letter of arr[1].toLowerCase()) {
+    if (arr[0].toLowerCase().indexOf(letter) === -1) {
```
Comment (tkachukivan) [inline tasks/21_check-letters/check-letters.js]: you can use `includes` method instead of `indexOf`

File: tasks/23_seek-destroy/seek-destroy.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export function destroyer(arr, ...args) {
+  return arr.filter((item) => args.indexOf(item) === -1 );
```
Comment (tkachukivan) [inline tasks/23_seek-destroy/seek-destroy.js]: `includes` :)

File: tasks/24_index-to-insert/index-to-insert.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export function getIndexToInsert(arr, valueToInsert) {
+  return arr.sort((a, b) => a-b)
```
Comment (tkachukivan) [inline tasks/24_index-to-insert/index-to-insert.js]: spaces `a - b `

File: tasks/25_rot13-encoder/rot13-encoder.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+export function rot13Encoder(str) {
+  return str.split('')
+      .map((letter) => {
+        if (letter.charCodeAt() >=65 && letter.charCodeAt() <= 77) {
```
Comment (tkachukivan) [inline tasks/25_rot13-encoder/rot13-encoder.js]: you use multiply times `letter.charCodeAt()` better to make a variable for this and use this variable

File: tasks/27_basic-subclasses/basic-subclasses.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+export class Woman extends Human {
+  constructor(name, sex) {
+    super(name, sex);
+  }
+}
+
+export class God {
+  static create() {
+    return [new Man('Adam', 'male'), new Woman('Eve', 'female')];
```
Comment (tkachukivan) [inline tasks/27_basic-subclasses/basic-subclasses.js]: it is better to pass `male` or `female` from `Woman/Man` class, as in your implementation I can easily make from `male` `female` or even `animal`

File: tasks/29_remove-duplicates/remove-duplicates.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+export function removeDuplicates(arr) {
+  let unique = [];
+  for (let item of arr) {
+    if (unique.indexOf(item) === -1 ) {
```
Comment (tkachukivan) [inline tasks/29_remove-duplicates/remove-duplicates.js]: `includes`
or cheat a bit and use `Set` :D to remove duplicates

File: tasks/30_people-in-bus/people-in-bus.js [hunk 7]
```diff
@@ -0,0 +1,5 @@
+export function peopleInBus(arr) {
+  let sum = 0;
+  arr.map((item) => (sum += item[0] - item[1]));
```
Comment (tkachukivan) [inline tasks/30_people-in-bus/people-in-bus.js]: don't you think that it is a great place to use `reduce` method ?

**Review by tkachukivan — CHANGES_REQUESTED** [review]

**Review by tkachukivan — APPROVED** [review]

File: tasks/22_remove-falsy-values/remove-falsy-values.js [hunk 8]
```diff
@@ -0,0 +1,4 @@
+export function removeFalsyValues(arr) {
+  return arr.filter((item) => Boolean(item));
```
Comment (mikhail-ev) [inline tasks/22_remove-falsy-values/remove-falsy-values.js:2]: arr.filter(Boolean)

**Review by mikhail-ev — APPROVED** [review]

**Pull request merged**